Makefile-ocp.mk

```diff
@@ -0,0 +1,12 @@
+include Makefile
+
+.PHONY: verify-ocp
+verify-ocp: manifests
```

Comment:
so IIUIC we are going to call both verify and verify-ocp separately in o/release?